### PR TITLE
Fileorbis integration

### PR DESCRIPTION
**Pull request by hakcekoce**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
FileOrbis Pack added

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      - '0'
+      - '1'
+      - '2'
+      required: true
+      secret: false
+    deprecated: false
+    description: Changes user status
+    execution: false
+    name: change-user-status
+    outputs:
```
Comment (daryakoval) [inline Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml:61]:
```suggestion
    outputs:
    - contextPath: FileOrbis.UserStatus.UserID
      description: User Id to change its status.
      type: String
```

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    status: int = int(args.get('status'))  # type:ignore
+
+    client.login()
+    result = client.change_user_status(user_id=user_id, status=status)
+    client.logout()
+
+    return CommandResults(
+        readable_output=result.get("Message"),
+        outputs=result,
+    )
```
Comment (daryakoval) [inline Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py:131]:
```suggestion
    return CommandResults(
        readable_output=result.get("Message"), 
        outputs=result,
        outputs_prefix='FileOrbis.UserStatus',
        outputs_key_field='UserID',
        raw_response=result
    )
```

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+
+def change_user_status_command(client: FileOrbisClient, args: Dict[str, Any]) -> CommandResults:
+    user_id: str = args.get('user_id')  # type:ignore
+    status: int = int(args.get('status'))  # type:ignore
+
+    client.login()
+    result = client.change_user_status(user_id=user_id, status=status)
+    client.logout()
+
```
Comment (daryakoval) [inline Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py:127]:
```suggestion
result['UserID'] = user_id
```

**Review by daryakoval — CHANGES_REQUESTED** [review]

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests (auto formatted)
+fromversion: 5.5.0
```
[the remark this answers is not in the document]
Comment (husrevbeyazisik) [reply Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml]: done

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      - '1'
+      - '2'
+      required: true
+      secret: false
+    deprecated: false
+    description: Changes user status
+    execution: false
+    name: change-user-status
+    outputs:
+    - contextPath: FileOrbis.Success
```
[the remark this answers is not in the document]
Comment (husrevbeyazisik) [reply Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml]: done

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+      description: True if operation completed successfully
+      type: Boolean
+    - contextPath: FileOrbis.Status
+      description: Result code of the operation
+      type: Number
+    - contextPath: FileOrbis.Message
```
[the remark this answers is not in the document]
Comment (husrevbeyazisik) [reply Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml]: done

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml (hunk 1 above)
Comment (husrevbeyazisik) [reply Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.yml:61]: done

File: Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py (hunk 2 above)
Comment (husrevbeyazisik) [reply Packs/FileOrbis/Integrations/FileOrbis/FileOrbis.py]: done

File: Packs/FileOrbis/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+FileOrbis content pack enables you to manage users.
+
+## What does this pack do?
```
Comment (mchasepan) [inline Packs/FileOrbis/README.md:3]:
```suggestion
##### What does this pack do?
```

File: Packs/FileOrbis/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+
+The integration in this pack enables you to:
+- Change of the user status
+
+## Authorization
```
Comment (mchasepan) [inline Packs/FileOrbis/README.md:8]:
```suggestion
##### Authorization
```

File: Packs/FileOrbis/README.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+FileOrbis content pack enables you to manage users.
+
+## What does this pack do?
+
+The integration in this pack enables you to:
+- Change of the user status
```
Comment (mchasepan) [inline Packs/FileOrbis/README.md:6]: What else does it do? Why would a user want to do that? Can you explain the use case better?

File: Packs/FileOrbis/README.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+FileOrbis content pack enables you to manage users.
```
Comment (mchasepan) [inline Packs/FileOrbis/README.md:1]: Do you want to explain what fileOrbis is? its value, etc.?

Comment (mchasepan) [reply Packs/FileOrbis/README.md:1]: Also include links to your site, etc. as desired.

File: Packs/FileOrbis/README.md (hunk 9 above)
Comment (hakcekoce) [reply Packs/FileOrbis/README.md]: Explanation in this file updated

**Pull request merged**